### ISSUE TITLE
V33 Theme download fails on android

### ISSUE THREAD

**Issue by bennettpeter** [MythTV/mythtv#715]
* **Platform**:  Andriod (fire-tv 4K)

* **MythTV version**: v33.1

* **Package version**:  mythfrontend-20230206-arm-v33.1.apk

* **Component**: mythfrontend

### What steps will reproduce the bug?
Download any theme

### How often does it reproduce? Is there a required condition?
Always

### What is the expected behaviour?
Theme download and install

### What do you see instead?
Message "ERROR: Downloading theme package from master backend"

### Additional information

file /data/data/org.mythtv.mythfrontend/files/tmp/Steppes-2.54_33.zip ahs actually been downloaded to the android device but the application is giving a failure message and failing to install it.

**Comment by mspieth** [comment]
Yes  I know. Its a problem with openssl 1.1.1 not being recognized. This is my current task but its proven harder than I expected.
There is also a patch to get trunk if the version does not exist for those running pre versions like me.

**Comment by bennettpeter** [comment]
Thanks for the update. 
I see the Steppes-2.54_33.zip file that was downloaded is only  336 bytes long, with an html page saying the file has moved, so it is garbage.
Native android uses SSL extensively, but it would be a lot of work to link into that.
I wonder if other parts of mythfrontend are affected by the lack of SSL.

**Comment by mspieth** [comment]
I have a solution after many weeks. Updates to packages and mythtv coming soon. Android QT doesnt seem to load the system CA certs automatically.

**Comment by mspieth** [comment]
I have pushed to mythtv and packaging. This issue should now be fixed.
Please close once tested.

**Comment by bennettpeter** [comment]
I tested #720 successfully on master. Please cherry-pick the required commits into fixes/33 so that I can test it here.

**Comment by mspieth** [comment]
Both repos cherry picked to fixes/33
